routes/v1.js

```diff
@@ -126,34 +147,25 @@ api.declare({
 /** Create a hook **/
 api.declare({
   method:       'put',
-  route:        '/hooks/:hookGroup/:hookId',
+  route:        '/hooks/:hookGroupId/:hookId',
   name:         'createHook',
   idempotent:   true,
-  scopes:       [["hooks:modify-hook:<hookGroup>/<hookId>"]],
+  scopes:       [["hooks:modify-hook:<hookGroupId>/<hookId>"]],
   input:        'create-hook-request.json',
   output:       'hook-definition.json',
   title:        'Create a hook',
   description: [
-    "Define and create a hook that will spawn a task when triggered. This",
-    "hook can be triggered through the web endpoint, or through a message",
-    "on the Pulse exchange that the hook is binded to."
+    "This endpoint will create a new hook.",
+    "",
+    "The caller's credentials need not satisfy `hook.task.scopes`. Instead,",
+    "the role for the hook must satisfy these scopes as well as the necessary",
+    "scopes to add the task to the queue.",
   ].join('\n')
 }, async function(req, res) {
-  var hookGroup = req.params.hookGroup;
+  var hookGroupId = req.params.hookGroupId;
   var hookId    = req.params.hookId;
   var hookDef   = req.body;
 
```

Comment:
I think we should use `deferAuth: true` and do the `if (!this.requires({hookGroupId, hookId})) { return; }` dance... 
Example: https://github.com/taskcluster/taskcluster-secrets/blob/da448a02e2639f7bb18ab4fc87bd69d0067cacd7/lib/api.js#L39-L41